pkg/doc/util/signature/ecdsa.go

```diff
@@ -45,6 +76,26 @@ func GetECDSASecp256k1Signer(privKey *ecdsa.PrivateKey) *ECDSASigner {
 	return &ECDSASigner{privateKey: privKey, PublicKey: &privKey.PublicKey, hash: crypto.SHA256}
 }
 
+// NewECDSASigner creates a new ECDSA signer based on the input elliptic curve.
+func NewECDSASigner(curve elliptic.Curve) (*ECDSASigner, error) {
```

Comment:
whats the plan for Ed25519?

Reply:
It's already present: https://github.com/hyperledger/aries-framework-go/blob/master/pkg/doc/util/signature/ed25519.go